src/network/types.rs

```diff
@@ -112,6 +112,10 @@ pub struct PerNetworkOptions {
     /// MAC address for the container interface.
     #[serde(rename = "static_mac")]
     pub static_mac: Option<String>,
+
+    /// Driver-specific options for this container.
+    #[serde(rename = "options", default)]
+    pub options: HashMap<String, String>,
```

Comment:
instead of using default I think it is cleaner to wrap this in an Option<>. While that makes accessing values a step more I think it carries the intend better that there can be empty.

Reply:
Agree, fixed.